src/main.ts

```diff
@@ -457,6 +457,16 @@ export function buildArgs(commands = new LazyCommands(), wrapMainHandler = wrapC
             + 'Use --assume-role-arn=no-role to override values in stack-args.yaml and use AWS_* env vars.')
         })
 
+        .option('color', {
+          type: 'string', default: 'auto',
+          choices: ['auto', 'always', 'never'],
+          description: description('whether to color output using ANSI escape codes')
+        })
+        .middleware((args) => {
+          if (args.color === 'never' || (args.color === 'auto' && ! process.stdout!.columns)) {
```

Comment:
Correct me if I'm wrong, but I don't believe the `!` after `stdout` is valid js

```suggestion
          if (args.color === 'never' || (args.color === 'auto' && ! process.stdout.columns)) {
```

Reply:
That’s a typescript feature that is the equivalent of `(process.stdout as any).columns`.

Reply:
I read it as the ‘trust me’ operator.

Reply:
I should add that I’d discourage its use on most Unbounce projects that have juniors working on them so there’s a case for using the verbose form even here.

Reply:
I see! My follow up question: Is the type assertion needed? 

The type of `process.stdout` is `WriteStream`, and `columns` is part of the interface.

Reply:
It was last time I worked on tty detection 2 years ago but I didn’t actually check this time. I’ll do that today.

Reply:
Hah, no it wasn't required on this. Removed.